cli/linter/schema.json

```diff
@@ -735,6 +735,9 @@
         },
         "rpc_pool_size": {
           "type": "integer"
+        },
+        "key_space_sync_interval": {
+          "type": "integer"
```

Comment:
I wonder if you want make it float, so tests will be faster?

Reply:
@buger that's a very good idea. I just pushed that change, tested and its ok